src/Npgsql/PostgresException.cs

```diff
@@ -87,6 +85,13 @@ void AddData<T>(string key, T value)
             }
         }
 
+        static string GetMessage(string sqlState, string messageText, int position, string? detail) =>
+            "ERROR " + sqlState + ": " + messageText +
```

Comment:
Do we really need the constant ERROR prefix, any specific reason to add it?

Reply:
Mostly, because that's more or less how PgAdmin shows that kind of errors - do you propose to drop it entirely, so the end result would look like:
```
some_error
Detail: some_detail
```

Reply:
I don't really care that much, but this message is already contained in an exception - so prefixing the message with ERROR seems unnecessary... We could also have an empty line separating the message from the detail/position if you think that makes sense.